### ISSUE TITLE
Missed calls show as 4 unread messages (should be 1?)

### ISSUE THREAD

**Issue by opusforlife2** [vector-im/element-android#3388]
#### Describe the bug

Element seems to be counting all the events that go into a missed call as unread messages in the chat list. The number that shows in the unread badge is '4'.

#### To Reproduce
1. Get a missed call.
2. Open chat list.
3. See the unread badge.

#### Expected behavior

The unread badge should contain just '1', as there has only been a single event from the user's perspective.

#### Smartphone (please complete the following information):
 - Device: Xiaomi Mi 5
 - OS: Android 10 (LOS 17.1)

#### Additional context
 - App version and store - 1.1.7 - F-Droid

**Comment by RSK-Robin** [comment]
Hello, I came across the same problem when using calls. I decided to look for a similar question and got here. It seems that the server is sending multiple events to the client. Each of which initially "unread". And since we are not on the chat screen, the events just accumulate and eventually we have several unread events from one missed call.

[Matrix specification VoIP](https://spec.matrix.org/v1.3/client-server-api/#client-behaviour-2)

**Comment by opusforlife2** [comment]
Yep. When you enable "show hidden events in the timeline", there are several events that occur around calls, which the app mistakenly shows as notifications.
